docs/content/Getting-Started/Cloud/06-Query-from-React.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Query from React app
+permalink: /getting-started/cloud/query-from-react-app
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 6
+---
+
+Cube offers both REST and GraphQL APIs, which can be used to query data from applications built in React or other frontend frameworks. 
+You can find your REST API endpoint on the “Overview” page. In development mode, Cube creates an isolated endpoint for testing data model changes without affecting production. 
+The structure of your REST API endpoint in development mode should follow the format below.
+
+```yaml
+https://<delpoyment-id>.<region>.cubecloudapp.dev/dev-mode/<dev-branch-name>/cubejs-api/v1
+```
+
+To test your REST API from your terminal, you can use [curl](https://curl.se/). Click on “How to connect your application” next to the REST API, and it will display a code snippet that you can run in your terminal to test the endpoint with curl.
```

Comment:
```suggestion
To test your REST API from your terminal, you can use [`curl`](https://curl.se/). Click on <Btn>How to connect your application</Btn> next to the REST API, and it will display a code snippet that you can run in your terminal to test the endpoint using `curl`.
```